algorithms/kernel/classifier/multiclass_confusion_matrix_dense_default_batch_container.h

```diff
@@ -37,39 +37,39 @@ namespace quality_metric
 {
 namespace multiclass_confusion_matrix
 {
-template<typename algorithmFPType, Method method, CpuType cpu>
-BatchContainer<algorithmFPType, method, cpu>::BatchContainer(daal::services::Environment::env *daalEnv)
+template <typename algorithmFPType, Method method, CpuType cpu>
+BatchContainer<algorithmFPType, method, cpu>::BatchContainer(daal::services::Environment::env * daalEnv)
 {
     __DAAL_INITIALIZE_KERNELS(internal::MultiClassConfusionMatrixKernel, method, algorithmFPType);
 }
 
-template<typename algorithmFPType, Method method, CpuType cpu>
+template <typename algorithmFPType, Method method, CpuType cpu>
 BatchContainer<algorithmFPType, method, cpu>::~BatchContainer()
 {
     __DAAL_DEINITIALIZE_KERNELS();
 }
 
-template<typename algorithmFPType, Method method, CpuType cpu>
+template <typename algorithmFPType, Method method, CpuType cpu>
 services::Status BatchContainer<algorithmFPType, method, cpu>::compute()
 {
-    Input *input = static_cast<Input *>(_in);
-    Result *result = static_cast<Result *>(_res);
-    Parameter *parameter = static_cast<Parameter *>(_par);
-    NumericTable *predictedLabelsTable   = static_cast<NumericTable *>(input->get(predictedLabels  ).get());
-    NumericTable *groundTruthLabelsTable = static_cast<NumericTable *>(input->get(groundTruthLabels).get());
+    Input * input                         = static_cast<Input *>(_in);
+    Result * result                       = static_cast<Result *>(_res);
+    Parameter * parameter                 = static_cast<Parameter *>(_par);
+    NumericTable * predictedLabelsTable   = static_cast<NumericTable *>(input->get(predictedLabels).get());
+    NumericTable * groundTruthLabelsTable = static_cast<NumericTable *>(input->get(groundTruthLabels).get());
 
-    NumericTable *confusionMatrixTable  = static_cast<NumericTable *>(result->get(confusionMatrix).get());
-    NumericTable *accuracyMeasuresTable = static_cast<NumericTable *>(result->get(multiClassMetrics).get());
+    NumericTable * confusionMatrixTable  = static_cast<NumericTable *>(result->get(confusionMatrix).get());
+    NumericTable * accuracyMeasuresTable = static_cast<NumericTable *>(result->get(multiClassMetrics).get());
 
-    daal::services::Environment::env &env = *_env;
-    __DAAL_CALL_KERNEL(env, internal::MultiClassConfusionMatrixKernel, __DAAL_KERNEL_ARGUMENTS(method, algorithmFPType),   \
```

Comment:
Seems / symbol is important for macro definition.

Reply:
It's just macro usage, not definition.